### PR TITLE
Removes thermal goggles from spawning in game

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request

Removes a pair of very overpowered items from two maps (Soro and Kut) as well as a RNG spawner. 

# Explain why it's good for the game

Items usually reserved for whiteout or other high-strength entities should not be available in normal gameplay. 

Vision and sightlines are a highly contentious area of balance, the game is carefully crafted around the idea that humans have limited visibility and they typically have to expend some effort or resource to increase that visiblity. Certain Marines have access to night vision, but this is either a rare kit (spec) or at greater cost (NVG optic that has a high cost + power drain).

Free thermal visors that not only provide night vision, but thermal vision, for free and forever, are highly overpowered in the hands of a marine. Normally reserved for Xenos, Predators or rare elite ERTs or whiteout operators, this is not a item I believe should be accessible to an average marine or survivor. 

(Don't forget this could be combined with binocs for extra long vision)

Furthermore this is a case of "the rich get richer", its more likely a veteran player will know of these items locations, ergo more experienced players will find it, and thus gain an ever stronger advantage of other players. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
maptweak: Removed in game access to thermal goggles from Kutjevo and Sorokyne, as well as removing a RNG spawner that can have them be spawned in. 
/:cl:

### CODE REVIEW

**Comment by c4xmaniac2** [comment]
This is something that's on more than half the maps in the game, and are intentionally placed. No reason to remove them, just melt them.

**Comment by Steelpoint** [comment]
Access to items that are typically reserved for Predators, whiteout operators or other highly dangerous entities for for survivors or marines is difficult to justify, especially considering regular marine access to night vision is a highly controlled and balanced factor. The fact these items have been in game for a long time does not grandfather them in, in my opinion. 

If someone had snuck in a HEAP magazine on a map for 5 years, I'd not take that as an excuse to leave the item in.

**Comment by Steelpoint** [comment]
I've done an extra investigation and removed additional thermal goggle spawns from Sorokyne and from a RNG spawner pool. Thank you for informing me of the additional spawns. I do not believe there are any other thermal goggles spawns left now.

**Comment by c4xmaniac2** [comment]
Alternate PR 

https://github.com/cmss13-devs/cmss13/pull/4780

**Comment by Diegoflores31** [comment]
> Access to items that are typically reserved for Predators, whiteout operators or other highly dangerous entities for survivors or marines is difficult to justify, especially considering regular marine access to night vision is a highly controlled and balanced factor. The fact these items have been in game for a long time does not grandfather them in, in my opinion.
> 
> If someone had snuck in a HEAP magazine on a map for 5 years, I'd not take that as an excuse to leave the item in.

First . Surv balance is not real and its been said by the devs numerous times 
second . CLF have access to thermals so its not a super rare impossible alien technology as you claim

File: code/game/objects/effects/spawners/random.dm [hunk 1]
```diff
@@ -268,9 +268,7 @@
 	icon_state = "loot_goggles"
 
 /obj/effect/spawner/random/goggles/item_to_spawn()
-	return pick(prob(4);/obj/item/clothing/glasses/thermal/syndi/bug_b_gone,\
-				prob(4);/obj/item/clothing/glasses/thermal/syndi,\
-				prob(4);/obj/item/clothing/glasses/thermal/monocle,\
+	return pick(prob(4);/obj/item/clothing/glasses/thermal/monocle,\
```
Comment (Diegoflores31) [inline code/game/objects/effects/spawners/random.dm:271]: monocle and eye patch are still thermals btw

**Comment by Steelpoint** [comment]
> > Access to items that are typically reserved for Predators, whiteout operators or other highly dangerous entities for survivors or marines is difficult to justify, especially considering regular marine access to night vision is a highly controlled and balanced factor. The fact these items have been in game for a long time does not grandfather them in, in my opinion.
> > If someone had snuck in a HEAP magazine on a map for 5 years, I'd not take that as an excuse to leave the item in.
> 
> First . Surv balance is not real and its been said by the devs numerous times second . CLF have access to thermals so its not a super rare impossible alien technology as you claim

ERTs are a bit of a odd bag since they get a weird mix of strong and weak gear.

But in the normal course of an average round no one has thermal vision outside of Predators and Xenos.

**Comment by Steelpoint** [comment]
Changes made and updated my reasoning.

**Comment by Zonespace27** [comment]
I've thought about this and decided to go with https://github.com/cmss13-devs/cmss13/pull/4780 for the following reasons:
- I do not believe that maps should be stripped of everything unique or powerful. We're not trying to make a competitive game where everything's balanced to perfection; it's okay to have unique and rare items that are powerful.
- Instead of removal, powerful items should be in hard-to-get locations, ideally with a level of randomness involved.

#4780 will need some changes, but I prefer its concept over this one.

**Pull request closed without merging**